locales/ru-RU.json

```diff
@@ -530,16 +530,16 @@
         "15": "Сверните на <STREET_NAMES> и сверните <TOWARD_SIGN>."
       },
       "ordinal_values": [
-        "1-м",
-        "2-м",
-        "3-м",
-        "4-м",
-        "5-м",
-        "6-м",
-        "7-м",
-        "8-м",
-        "9-м",
-        "10-м"
+        "первом",
+        "втором",
+        "третьем",
+        "четвёртом",
+        "пятом",
+        "шестом",
+        "седьмом",
+        "восьмом",
+        "девятом",
+        "десятом"
```

Comment:
very good, this person gets that "verbal" means you spell it out so text to speech can read it properly